tests/project/_common/test_movement.py

```diff
@@ -0,0 +1,437 @@
+"""Tests for tasks related to movement with Tamr projects"""
+import pytest
+from tamr_toolbox import utils, workflow
+from tests._common import get_toolbox_root_dir
+from tamr_toolbox.project import categorization
+from tamr_toolbox.models.project_artifacts import CategorizationArtifacts as catfacts
```

Comment:
Current tests only cover categorization. The other project types need to be tested as well. You might be able to use your existing unit tests by parameterizing the tests by project type using the `pytest.mark.parametrize` decorator. Example here:
https://github.com/Datatamer/tamr-toolbox/blob/develop/tests/data_io/test_csv.py#L75

Reply:
Thanks for the suggestion, very useful decorator!
Added tests for mastering, schema mapping, and golden records.

Reply:
Fantastic!